h/services/document.py

```diff
@@ -44,4 +44,4 @@ def fetch_by_groupid(self, groupid, userid=None):
 
 
 def document_service_factory(_context, request):
-    return DocumentService(request.db)
+    return DocumentService(session=request.db)
```

Comment:
When I wrote the test for this I think it's nicer to use kwargs than positional args, as they are more refactoring friendly.